test/runtime/kafka.go

```diff
@@ -129,9 +129,11 @@ var _ = Describe("RuntimeValidatedKafka", func() {
 		Expect(err).Should(BeNil())
 
 		endPoints, err := vm.PolicyEndpointsSummary()
-		Expect(err).Should(BeNil())
-		Expect(endPoints[helpers.Enabled]).To(Equal(1))
-		Expect(endPoints[helpers.Disabled]).To(Equal(2))
+		Expect(err).Should(BeNil(), "Cannot get policy enforcement summary")
+		Expect(endPoints[helpers.Enabled]).To(Equal(1),
+			"Kafka container does not have any policy enforcement, and it should")
```

Comment:
`"Check number of endpoints with policy enforcement enabled"`

The kafka context is clear from the test context.